src/annotations/components/EditAnnotationForm.tsx

```diff
@@ -0,0 +1,177 @@
+                onChange={e => handleEditAnnotationChange(e)}
+                status={ComponentStatus.Default}
+                size={ComponentSize.Medium}
+              />
+            </Form.Element>
+            <Form.Element
+              label="Summary"
+              className="edit-annotation-form-label"
+            >
+              <Input
+                name="summary"
+                value={editAnnotationState.summary}
+                onChange={handleEditAnnotationChange}
+                status={ComponentStatus.Default}
+                size={ComponentSize.Medium}
+              />
+            </Form.Element>
+            <Form.Element
+              label="Message"
+              className="edit-annotation-form-label"
+            >
+              <TextArea
+                name="message"
+                placeholder="Try writing markdown here..."
+                value={editAnnotationState.message}
+                onChange={handleEditAnnotationChange}
+                status={ComponentStatus.Default}
+                className="edit-annotation-form-textarea"
+              />
+            </Form.Element>
+          </Panel>
+        </Grid.Column>
+        <Grid.Column widthSM={Columns.Six} widthXS={Columns.Twelve}>
+          <h3 className="edit-annotation-header-text">Store Annotation</h3>
+          <Panel
+            backgroundColor={InfluxColors.Onyx}
+            className="edit-annotation-panel"
+          >
+            <Form.Element
+              label="Annotation Stream"
+              className="edit-annotation-form-label"
+            >
+              <Dropdown
+                button={(active, onClick) => (
+                  <Dropdown.Button
+                    active={active}
+                    onClick={onClick}
+                    testID="stream-selector--dropdown"
+                  >
+                    {annotationStreams[0].stream}
+                  </Dropdown.Button>
+                )}
+                menu={onCollapse => (
+                  <Dropdown.Menu
+                    onCollapse={onCollapse}
+                    testID="stream-selector--dropdown-menu"
+                  >
+                    {annotationStreams.map(a => a.stream)}
+                  </Dropdown.Menu>
+                )}
+              />
+            </Form.Element>
+          </Panel>
+        </Grid.Column>
+      </Grid>
+      <Overlay.Footer className="edit-annotation-form-footer">
+        <Button
+          text="Delete Annotation"
+          onClick={() => {}}
+          color={ComponentColor.Danger}
+          style={{marginRight: '15px'}}
+        />
+        <div className="edit-annotation-buttons">
+          <Button
+            text="Cancel"
+            onClick={onClose}
+            color={ComponentColor.Default}
+            className="edit-annotation-cancel"
+          />
+          <Button
+            text="Save Changes"
+            onClick={(e: FormEvent) =>
```

Comment:
Yeah this was a holdover from some other funky stuff - i'll give you a bit of context, was a fun ride - you can't use the callback form of useState with the [e.target.name]: [e.target.value] because React loses context to the synthetic event.... so was playing around with that for a bit - just thought its a cool tidbit to share